fitparser/src/profile/mod.rs

```diff
@@ -232,6 +230,27 @@ pub fn unknown_field(field_def_num: u8, value: Value) -> FitDataField {
     )
 }
 
+/// Create an "unknown" field as a placeholder if we don't have any field information
+pub fn unknown_field_possibly_dev_field(
```

Comment:
Once we're storing dev fields in a separate mapping I think we could leverage the same unknown_field function, the location in the data struct would remove the need for the alternate naming.

Reply:
this function was indeed removed by splitting up the fields and developer_fields again.